# patch to use ssh rather than rsh for TCP/IP socket communication
# in GAMESS' DDI_SOC interface
--- gamess/ddi/tools/ddikick/ddikick.c	2004-05-03 16:23:13.000000000 +0000
+++ gamess-gentoo/ddi/tools/ddikick/ddikick.c	2005-09-04 15:56:50.000000000 +0000
@@ -102,7 +102,7 @@
           remoteshell = argv[++i];
        } else {
           if((remoteshell = getenv("DDI_RSH")) == NULL) {
-              remoteshell = (char *) strdup("rsh");
+              remoteshell = (char *) strdup("ssh");
           }
        }
 
